doc/getting_started/getting_started.rst

```diff
@@ -216,10 +238,9 @@ Make sure to unset the ZEPHYR_SDK_INSTALL_DIR if you don't use the
 SDK's host tools. See `Building without the Zephyr SDK`_ for details.
 
 It is possible to use the Zephyr SDK's host tools along with a 3rd
-party cross compiler. To do that, keep the ZEPHYR_SDK_INSTALL_DIR
-environment variable set to the Zephyr SDK installation directory.
-See `Set Up the Development Environment`_ for more details on the
-ZEPHYR_SDK_INSTALL_DIR environment variable.
+party cross compiler. To do that, keep ZEPHYR_SDK_INSTALL_DIR set to
```

Comment:
Add the double back-ticks on the environment variable name:

    ``ZEPHYR_SDK_INSTALL_DIR``